.travis.yml

```diff
@@ -24,6 +24,7 @@ jdk:
   - oraclejdk8
 
 services:
+  - docker
```

Comment:
Docker image caching is a hot topic in Travis world :) It's not supported by Travis, however you can cache it manually with "docker export", "docker import". But! Travis devs were saying that it's actually slower than pulling from Docker Hub :D